builders/integrations/oracles/api3.md

```diff
@@ -0,0 +1,365 @@
+- **`data`**: In case of a successful response, this is the requested data which has been encoded and contains a timestamp in addition to other response data. Decode it using the `decode()` function from the `abi` object.
+
+!!! note
+    Sponsors should not fund a `sponsorWallet` with more then they can trust the Airnode with, as the Airnode controls the private key to the `sponsorWallet`. The deployer of such Airnode undertakes no custody obligations, and the risk of loss or misuse of any excess funds sent to the `sponsorWallet` remains with the sponsor.
+
+### [Try deploying it on Remix!](https://remix.ethereum.org/#url=https://github.com/vanshwassan/RemixContracts/blob/master/contracts/Requester.sol&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.9+commit.e5eed63a.js)
+
+
+### Contract Addresses
+
+=== "Moonbeam"
+    | Contract  |          Addresses         |
+    |:-----------:|:-----------------------------------------------------:|
+    | AirnodeRrpV0 | `{{ networks.moonbeam.api3.rrp }}` |
+
+=== "Moonriver"
+    |  Contract  |           Addresses          |
+    |:------------:|:----------------------------------------------------:|
+    | AirnodeRrpV0 | `{{ networks.moonriver.api3.rrp }}`  |
+
+
+=== "Moonbase Alpha"
+    |  Contract  |          Addresses           |
+    |:------------:|:-----------------------------------------------------:|
+    | AirnodeRrpV0  | `{{ networks.moonbase.api3.rrp }}`  |
+
+
+## Using dAPIs - API3 Datafeeds {: #dapis}
+
+[dAPIs](https://docs.api3.org/explore/dapis/what-are-dapis.html) are continuously updated streams of off-chain data, such as the latest cryptocurrency, stock and commodity prices. They can power various decentralized applications such as DeFi lending, synthetic assets, stablecoins, derivatives, NFTs and more.
+
+The data feeds are continuously updated by [first-party oracles](https://docs.api3.org/explore/introduction/first-party.html) using signed data. dApp owners can read the on-chain value of any dAPI in realtime.
+
+Due to being composed of first-party data feeds, dAPIs offer security, transparency, cost-efficiency and scalability in a turn-key package.
+
+The [API3 Market](https://market.api3.org/dapis) enables users to connect to a dAPI and access the associated data feed services.
+
+![API3 Remix deploy](/images/builders/integrations/oracles/api3/SS4.png)
+
+[*To know more about how dAPIs work, click here*](https://docs.api3.org/explore/dapis/what-are-dapis.html)
+
+<!-- ### Types of dAPIs
+
+#### Self-funded dAPIs
+Self-funded dAPIs offer developers the opportunity to experience data feeds with
+minimal up-front commitment, providing a low-risk option prior to using a
+managed dAPIs.
+
+#### Managed dAPIs
+Managed dAPIs are sourced from multiple first-party oracles and aggregated using
+a median function. Compared to self-funded dAPIs, **managed dAPIs are monetized**,
+as API3 requires payment in USDC on Ethereum Mainnet to operate them. -->
```

Comment:
@vanshwassan just wanted to double check on this section, should it be commented out?

Reply:
Yeah, this can be commented out 👍🏻